app/assets/stylesheets/govuk_publishing_components/components/_breadcrumbs.scss

```diff
@@ -13,16 +13,16 @@
   @include breadcrumbs;
   @include touch-friendly-links();
 
-  .breadcrumb-for-current-page {
+  .gem-c-breadcrumbs--current {
     color: $secondary-text-colour;
     text-decoration: none;
   }
 
   @include media-down(mobile) {
-    &.collapse-on-mobile li {
+    &.gem-c-breadcrumbs--collapse-on-mobile li {
```

Comment:
We tend to avoid selecting by element type - I'd suggest replacing `li` with a class, e.g: `gem-c-breadcrumbs__item` or `gem-c-breadcrumbs__list-item`

Reply:
You'd then probably be able to make [some of the test selectors](https://github.com/alphagov/govuk_publishing_components/pull/309/commits/0082e890413d19d2077a58b9016715d07293f411#diff-79d1370df22d4cef0c4e9a10d443b147R32) more specific